openlibrary/plugins/openlibrary/js/search.js

```diff
@@ -78,7 +80,7 @@ export function initSearchFacets(facetsElem) {
                 facetsElem.replaceWith(newFacetsElem)
                 hydrateFacets()
 
-                document.title = data.title
+                setTitle(data.title)
```

Comment:
Can we avoid html-escaping the title entirely on the python side, where `data` is created? Then we can do `document.title = data.title`.